### PR TITLE
SEO-172844-WPF-weblink-validatetor-image

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes in image name ,fold and Alt text. Please review.
Thanks.
Regards,
Edith

### CODE REVIEW

File: wpf/Color-Picker-Palette/Getting-Started.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ public partial class MainWindow : Window {
 {% endcapture %}
 {{ codesnippet2 | OrderList_Indent_Level_1 }}
 
-![ColorPickerPalette control added by xaml and code](Getting-Started_images/Getting-Started_Designer.png)
+![WPF Color Picker Palette control added by xaml and code](getting-started_images/wpf-color-picker-palette-xaml-code-control.png)
```
Comment (MallikaRK) [inline wpf/Color-Picker-Palette/Getting-Started.md:118]: change like that
wpf-color-picker-palette-control-added-via-xaml-code

File: wpf/Color-Picker-Palette/Getting-Started.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ colorPickerPalette.Height = 40;
 {% endhighlight %}
 {% endtabs %}
 
-![ColorPickerPalette programmatically picked the red color](Dealing-with-ColorPickerPalette_images/Colorprogrammatically.png)
+![WPF  Color Picker Palette programmatically picked the red color](dealing-with-color-picker-palette_images/wpf-programmatically-color.png)
```
Comment (MallikaRK) [inline wpf/Color-Picker-Palette/Getting-Started.md:146]: Don't change image folder name. just update casing alone.
image
Use proper image folder name.

File: wpf/Color-Picker-Palette/Getting-Started.md [hunk 3]
```diff
@@ -173,7 +173,7 @@ colorPickerPalette.Height = 40;
 {% endhighlight %}
 {% endtabs %}
 
-![ColorPickerPalette control with theme and standard color items](Getting-Started_images/ThemeColorPanel.png)
+![WPF Color Picker Palette control with theme and standard color items](getting-started_images/wpf-color-picker-palette-theme-standard-items.png)
```
Comment (MallikaRK) [inline wpf/Color-Picker-Palette/Getting-Started.md:176]: wpf-color-picker-palette-with-standard-theme
Use the image name here.

File: wpf/Color-Picker-Palette/Getting-Started.md [hunk 4]
```diff
@@ -288,7 +288,7 @@ colorPickerPalette.Height = 40;
 {% endhighlight %}
 {% endtabs %}
 
-![ColorPickerPalette with more color panel](Getting-Started_images/MoreColorWindow.gif)
+![WPF Color Picker Palette with more color panel](getting-started_images/wpf-color-picker-palette-color-window.gif)
```
Comment (MallikaRK) [inline wpf/Color-Picker-Palette/Getting-Started.md:291]: twice color present in image name.
wpf-color-picker-palette-window 
use the above

File: wpf/Color-Picker-Palette/Working-with-ColorPickerPalette.md [hunk 5]
```diff
@@ -204,7 +204,7 @@ colorPickerPalette.Height = 40;
 {% endhighlight %}
 {% endtabs %}
 
-![ColorPickerPalette with black and white color variants](Dealing-with-ColorPickerPalette_images/BlackWhite.png)
+![WPF Color Picker Palette with black and white color variants](dealing-with-color-picker-palette_images/wpf-color-variants.png)
```
Comment (MallikaRK) [inline wpf/Color-Picker-Palette/Working-with-ColorPickerPalette.md:207]: Image folder name is wrong. check my previous comments. Update in all places.

File: wpf/Color-Picker-Palette/Getting-Started.md (hunk 1 above)
Comment (Edithabich) [reply wpf/Color-Picker-Palette/Getting-Started.md:118]: resolved

File: wpf/Color-Picker-Palette/Getting-Started.md (hunk 2 above)
Comment (Edithabich) [reply wpf/Color-Picker-Palette/Getting-Started.md:146]: resolved

File: wpf/Color-Picker-Palette/Getting-Started.md (hunk 3 above)
Comment (Edithabich) [reply wpf/Color-Picker-Palette/Getting-Started.md:176]: resolved

File: wpf/Color-Picker-Palette/Getting-Started.md (hunk 4 above)
Comment (Edithabich) [reply wpf/Color-Picker-Palette/Getting-Started.md:291]: Resolved

File: wpf/Color-Picker-Palette/Working-with-ColorPickerPalette.md (hunk 5 above)
Comment (Edithabich) [reply wpf/Color-Picker-Palette/Working-with-ColorPickerPalette.md:207]: resolved

File: wpf/Color-Picker-Palette/Getting-Started.md (hunk 2 above)
Comment (Edithabich) [reply wpf/Color-Picker-Palette/Getting-Started.md:146]: resolved

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**